capstone/src/main/java/database/DatabaseRetriever.java

```diff
@@ -0,0 +1,102 @@
+package database;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.EntityNotFoundException;
+import com.google.appengine.api.datastore.KeyFactory;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.sps.graph.UserVertex;
+
+public class DatabaseRetriever implements Serializable {
+
+  /** */
+  private static final long serialVersionUID = 1L;
```

Comment:
I'm not sure this is used anywhere? If it's necessary, then please fill out the comment because I don't know what it does as well as make it a constant since it's final.

Reply:
I believe this has something to do with being a serializable object.

https://www.baeldung.com/java-serial-version-uid#serial-version-uid

If you're saving data into some place, and then updating your logic, and trying to load that serialized data back, you could end up in a mess if they objects weren't the same. This versioning allows you to identify those changes.